src/main/java/net/spals/midas/GoldFile.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2015, James T Spagnola & Timothy P Kral
+ * Copyright (c) 2016, James T Spagnola & Timothy P Kral
```

Comment:
Just as a side note, I think there's a header plugin that helps with this sort of thing.